### ISSUE TITLE
Add dns.seby.io

### ISSUE THREAD

**Issue by publicarray** [privacytoolsIO/privacytools.io#1734]
## Basic Information

**Name:** dns.seby.io
**Filtering:** private IPs for DNS Rebind protection
**Privacy policy:** No Logging just aggregate stats: https://dns.seby.io/stats.html
**Protocols:** DoH, DoT & DNSCrypt
**Server/Location:** Sydney, Australia
**Source:** https://github.com/publicarray/dns-resolver-infra
**Type:** hobby project
**Website:** https://dns.seby.io/

## Description

There where few Australian servers so I deployed one in early 2017

### Required features:

* [x] supports DoH or DoT (please check the port numbers the servers have different ports due to historic reasons)
* [x] supports DNSSEC
* [x] doesn't log IP addresses during normal operation

#### Desired features

* [x] supports QNAME minimization

```
❯ dig +short txt qnamemintest.internet.nl @127.0.0.1
a.b.qnamemin-test.internet.nl.
"HOORAY - QNAME minimisation is enabled on your resolver :)!"
```

## My connection with the software

Author and enthusiastic early adopter

**Comment by freddy-m** [comment]
@privacytools/editorial I'm in favour of adding this.

**Comment by Baker-ux-dot** [comment]
> **Privacy policy:** No Logging just aggregate stats: https://dns.seby.io/stats.html

The privacy policy is 404.

**Comment by publicarray** [comment]
Yes because I don't collect aggregate stats anymore. Collecting data got a bit to complicated with docker and 2 servers and used CPU/memory that could be used to serve actual requests.
